### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by DeastinY**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,45 +19,44 @@ Murmur is not compiled with gRPC support by default (as of 1.3.0). If you are us
...
     ```yaml
     # A list of application service config files to use
-    #
     app_service_config_files:
-    - mumble-config.yaml
+    - /etc/matrix-synapse/mumble-registration.yaml
     ```
 
-4. Fill out `mumble-config.yaml`
+4. Create a file `mumble-config.yaml`
```
Comment (mymindstorm) [inline README.md:47]: How about this?
```suggestion
4. Create a new file named `mumble-config.yaml` on the server that `matrix-appservice-mumble` is installed on.
```

File: README.md [hunk 2]
```diff
@@ -21,45 +19,44 @@ Murmur is not compiled with gRPC support by default (as of 1.3.0). If you are us
...
 
     - Look at the [mumble-config.yaml.example](https://github.com/mymindstorm/matrix-appservice-mumble/blob/master/mumble-config.yaml.example) file for an example
```
Comment (mymindstorm) [inline README.md:49]:
```suggestion
    - Copy the contents of [mumble-config.yaml.example](https://github.com/mymindstorm/matrix-appservice-mumble/blob/master/mumble-config.yaml.example) into `mumble-config.yaml` and change the options as needed.
```

File: README.md [hunk 3]
```diff
@@ -21,45 +19,44 @@ Murmur is not compiled with gRPC support by default (as of 1.3.0). If you are us
...
 
     - `matrixRoom` should be a private room
         
-        1. Create a new invite-only room
+        1. Create a new room (should be invite-only, which is the default)
 
         2. Invite `@mumblebot:<your homeserver domain>` to the room. Riot will warn you that the user does not exist, click "Invite anyway"
```
Comment (mymindstorm) [inline README.md:55]:
```suggestion
        2. Invite `@mumblebot:<your homeserver domain>` to the room. Your client may warn you that the user does not exist, click "Invite anyway"
```

File: README.md [hunk 4]
```diff
@@ -21,45 +19,44 @@ Murmur is not compiled with gRPC support by default (as of 1.3.0). If you are us
...
 
         3. Copy the internal room id of the newly created room to `mumble-config.yaml`
 
-5. `matrix-appservice-mumble -c ./mumble-config.yaml -f ./mumble-registration.yaml`
+5. `matrix-appservice-mumble -c /path/to/mumble-config.yaml -f /path/to/mumble-registration.yaml`
```
Comment (mymindstorm) [inline README.md:59]:
```suggestion
5. Start the bridge

```bash
$ matrix-appservice-mumble -c /path/to/mumble-config.yaml -f /path/to/mumble-registration.yaml`
```

**Review by mymindstorm — CHANGES_REQUESTED** [review]
Thanks for the suggestions!

File: README.md [hunk 5]
```diff
@@ -1,10 +1,8 @@
 # matrix-appservice-mumble
 
-A simple Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.
+A Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.
 
-## Install
-
-### These instructions are for a development version, please use the instructions located [here](https://github.com/mymindstorm/matrix-appservice-mumble/tree/v0.2.0#install)
```
Comment (mymindstorm) [inline README.md:7]: Also, could you add this back and have it point to [here, where I have backported this](https://github.com/mymindstorm/matrix-appservice-mumble/tree/0.2.0-install-instructions#installation)?

File: README.md [hunk 6]
```diff
@@ -1,10 +1,8 @@
 # matrix-appservice-mumble
 
-A simple Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.
```
[the remark this answers is not in the document]
Comment (DeastinY) [reply README.md:3]: :stuck_out_tongue_winking_eye:

File: README.md (hunk 5 above)
Comment (DeastinY) [reply README.md:7]: alrighty x)

**Review by mymindstorm — APPROVED** [review]

**Pull request merged**